src/components/search/SearchSuggestions.tsx

```diff
@@ -66,13 +54,12 @@ const SearchSuggestions: FunctionComponent<SearchSuggestionsProps> = ({ parentId
 
             <div className='searchSuggestionsList padded-left padded-right'>
                 {suggestions.map(item => (
-                    <div
```

Comment:
This removes the wrapping `<div>` which does change the look of search suggestions, but I believe the new look is what was intended.

Reply:
The previous appearance was intentional so I rolled this change back.